src/viur_cli/version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "1.1.1"
+__version__ = "1.1.2"
```

Comment:
This huge change is NOT a patch!
```suggestion
__version__ = "2.0.0"
```

Reply:
You should consider to split development of viur-cli into
- main branch for current stable
- develop branch for upcoming version 2

And release this stuff at first as 2.0.0.dev1 so that projects can test it. Bugfixes still remain in main, and are merged into develop. See viur-core development, which exactly makes it this way.